src/components/SignerPanel/SignerPanel.js

```diff
@@ -115,6 +115,7 @@ class SignerPanel extends React.PureComponent {
 
     return {
       intent: (transaction && this.transactionIntent(bag)) || {},
+      external: transaction.external,
```

Comment:
Rather than encoding this information into the transaction object aragon.js sends, we could do a check here with `transaction.to` against the list installed apps to detect if it's an external transaction.